src/legacy/core_plugins/kibana/public/discover/_discover.scss

```diff
@@ -169,7 +169,7 @@ discover-app {
 
 // SASSTODO: replace the margin-right value with a variables
 .dscField__icon {
-  margin-right: 5px;
+  margin-right: 10px;
```

Comment:
Since you're changing this, can you go ahead and take on that todo? Likely if you're looking for a bit more space `$euiSizeS` will work. It'll get you to 8px.

Reply:
Of course, we should further adapt this component, icons etc. but in a separate PR.

Reply:
i will adapt this change of course, more margin here is necessary, because react removed the whitespace angular added ... further work will include taking care of unused classes (for scripted fields e.g., guess there should be an icon for that  etc.)